contracts/sweep/BaseTokenSweep.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+/**
+ * @title Abstract upgradeable contract that provides the ability to sweep erroneous transferred tokens to a defined beneficiary address
+ *
+ * @dev Access control implementation is required for many functions by design
+ */
+abstract contract BaseTokenSweep is UUPSUpgradeable {
```

Comment:
question: what is base about the token sweep?

Why is this not simply `TokenSweep` or `TokenSweeper`?